MetovaTestKit.xcodeproj/project.pbxproj

```diff
@@ -612,13 +612,16 @@
 			files = (
 			);
 			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
```

Comment:
What's this?

Reply:
I am not sure. Xcode 10 added this maybe ? or pod install